elastica/dissipation.py

```diff
@@ -0,0 +1,117 @@
+__doc__ = """ Built in dissipation module implementations """
+__all__ = [
+    "DamperBase",
+    "ExponentialDamper",
+]
+
+from typing import Type, Union
+import numpy as np
+from abc import ABC, abstractmethod
+from elastica.rod import RodBase
+from elastica.rigidbody import RigidBodyBase
+
+
+class DamperBase(ABC):
```

Comment:
@bhosale2 You sure ABC is the right one you wanna use here?

Reply:
The idea here was to design the `damping` module the same way as `constraints`, where an abstract class with an abstract method `dampen_rates()` needs to be defined. @skim0119 if you feel I'm missing something here following the way things are done in `pyelastica` let me know. Or better please suggest the alternative you have in mind.